discord/flags.py

```diff
@@ -719,10 +719,20 @@ def members(self):
 
     @flag_value
```

Comment:
`alias_flag_value` or something should be used for either this one or moderation, I'm not entirely sure how it works

Reply:
`alias_flag_value` makes it so the attr is ignored when iterating through flags, so I'm inclined to believe it should be on the alias (`bans`) which I'll do, but there seems to be some inconsistency. For the most part it's used on aliases/shortcut intents, but it appears to be swapped around for the `emojis_and_stickers` intent.

https://github.com/Pycord-Development/pycord/blob/master/discord/flags.py#L743
```py
    @flag_value
    def emojis(self):
        """:class:`bool`: Alias of :attr:`.emojis_and_stickers`.

        .. versionchanged:: 2.0
            Changed to an alias.
        """
        return 1 << 3

    @alias_flag_value
    def emojis_and_stickers(self):
        """:class:`bool`: Whether guild emoji and sticker related events are enabled.

        .. versionadded:: 2.0

        This corresponds to the following events:
        ..."""
```